File: Unidad_3/Java/Programa13_U3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.programa13_u3;

/**
 *
 * @author LaboratorioLTI2
 */
import java.util.Scanner;
public class Programa13_U3 {

    public static void main(String[] args) {
        Scanner sc = new Scanner (System.in);
        int c, num;
        for(c=1;c<=100;c++)
        {
        System.out.println("Ingresa los valores totales ");
        num=sc.nextInt();
        
        if (num>0)
        {
        
         System.out.println("Es positivo ");
        }else
        {
         System.out.println("Es negativo ");
        }
        }
    }
}

File: Unidad_3/C#/Programa13_U3.cs
using System;

namespace Programa13_U3
{
    class Program
    {
        static void Main(string[] args)
        {
            int c, num;
            for (c = 1; c <= 100; c++)
            {
                Console.WriteLine("Ingresa los valores totales: ");
                num = int.Parse(Console.ReadLine());

                if (num > 0)
                {
                    Console.WriteLine("Es positivo");
                }
                else
                {
                    Console.WriteLine("Es negativo");
                }
            }
        }
    }
}
